Ansible/ansible_collections/jfrog/platform/roles/artifactory_nginx/tasks/Debian-ownstream.yml

```diff
@@ -0,0 +1,25 @@
+- name: ensure apt-transport-https is installed
+  anible.builtin.napt:
```

Comment:
This  `anible.builtin.napt:` doesn't work

Reply:
b3e45d517d2671592ce51008483f98768d579139